spring-boot-project/spring-boot/src/main/java/org/springframework/boot/webservices/client/WebServiceTemplateBuilder.java

```diff
@@ -520,27 +520,28 @@ private WebServiceMessageSenders(boolean onlyAdditional, Set<WebServiceMessageSe
 			this.messageSenders = messageSenders;
 		}
 
-		public boolean isOnlyAdditional() {
+		boolean isOnlyAdditional() {
```

Comment:
I don't think we've yet defined a good code style for these kinds of methods. Obviously there's nothing wrong with dropping `public` but I don't think we're very consistent across the codebase. We should probably come up with a standard and start to enforce it with checkstyle.

Reply:
We discussed this today and like the style that you've adopted. Going forward, we'll not have unnecessary public methods.

Reply:
Thanks